assets/html/scss/documenter/layout/_search.scss

```diff
@@ -14,5 +14,59 @@
     .docs-highlight {
       background-color: yellow;
     }
+
+    .search-result-link {
+      border-radius: 0.7em;
+      transition: all 300ms;
+    }
+    
+    .search-result-link:hover, .search-result-link:focus {
+      background-color: $search-result-link-hover;
+    }
+    
+    .search-result-link .property-search-result-badge {
+      transition: all 300ms;
+    }
+    
+    .property-search-result-badge {
+      padding: 0.15em 0.5em;
+      font-size: 0.8em;
+      font-style: italic;
+      text-transform: none !important;
+      line-height: 1.5;
+      color: $search-result-badge-color;
+      background-color: $search-result-badge-background-color;
+      border-radius: 0.6rem;
+    }
+    
+    .search-result-link:hover .property-search-result-badge, .search-result-link:focus .property-search-result-badge {
+      color: $search-f1f5f9;
+      background-color: $search-333;
+    }
+    
+    .search-result-highlight {
+      background-color: $search-result-highlight;
+      color: black !important;
```

Comment:
Are these `!important`s important? I think ideally we should avoid them, unless we really need them. But this can always be "fixed" later as well.